rt-scripts-optimizer.php

```diff
@@ -14,15 +14,15 @@
  */
 
 // Include settings options page.
-require_once 'includes/settings-page.php';
+require_once plugin_dir_path( __FILE__ ) . 'includes/settings-page.php';
```

Comment:
I'd suggest defining constant for base plugin folder path and then used it and it can be used anywhere in plugin.